src/m365/commands/login.ts

```diff
@@ -93,9 +113,9 @@ class LoginCommand extends Command {
           break;
         case 'certificate':
           auth.connection.authType = AuthType.Certificate;
-          auth.connection.certificate = args.options.certificateBase64Encoded ? args.options.certificateBase64Encoded : fs.readFileSync(args.options.certificateFile as string, 'base64');
+          auth.connection.certificate = getCertificate(args.options);
```

Comment:
I wonder if someone did specify the cert file or the encoded string in the `setup` command wouldn't it be kinda misleading we still need to provide the `--authType certificate` when login? Maybe when those configs are set this should become the default login method? 
What is your take on that?

Reply:
Good point, will update